pytket/docs/display.rst

```diff
@@ -1,5 +1,22 @@
 pytket.circuit.display
 ==================================
 
+Contains several functions for rendering interactive circuit diagrams.
+
+.. note:: Rendering circuits with ``pytket.circuit.display`` requires an internet connection. Using the pytket circuit renderer offline can be done by installing the `pytket-offline-renderer extension <https://github.com/CQCL/pytket-offline-renderer>`_.
+
+    ::
+
+        pip install pytket-offline-renderer
+
 .. automodule:: pytket.circuit.display
-    :members: render_circuit_jupyter, render_circuit_as_html
+    :members: render_circuit_jupyter, render_circuit_as_html, view_browser
+
+.. jupyter-execute::
+
+    from pytket import Circuit
+    from pytket.circuit.display import render_circuit_jupyter
```

Comment:
Do you want to explain how to use the offline renderer too? (`from pytket.extensions.offline_display import render_circuit_jupyter`)

Reply:
Ah yes I intended to do this but forgot to push the last commit I had locally. Will do that now.